src/spec-node/collectionCommonUtils/generateDocsCommandImpl.ts

```diff
@@ -191,8 +186,36 @@ async function _generateDocumentation(
                 }
 
                 // Write new readme
-                fs.writeFileSync(readmePath, newReadme);
-            }
+            fs.writeFileSync(readmePath, newReadme);
```

Comment:
Use async variant to allow other code to make progress while this is waiting for completion.